marketstack/src/endpoint/eod.ts

```diff
@@ -0,0 +1,42 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest, Config } from '@chainlink/types'
+import { DEFAULT_INTERVAL, DEFAULT_LIMIT } from '../config'
+import { util } from '@chainlink/ea-bootstrap'
+
+export const Name = 'eod'
+
+const customError = (data: any) => data.Response === 'Error'
```

Comment:
@krebernisak Should we be adding typing for each API response shape? Seems like it might be hard to maintain, but could provide valuable debugging information.

Reply:
Not in this PR to keep it fast & simple.

But in the future, we should type anything that's useful, and I think that will become useful at one point. The further we go with composite adapters, the more we use the data provider adapters programmatically, the more useful we are going to find those types.

First, we should finish this migration, then focus on improvements to our internal framework data types, and then we can make another push like this where we add a bunch of underlying API data types in one push. At that point it could start being useful :)